modules/gapi/src/executor/gstreamingexecutor.hpp

```diff
@@ -202,14 +193,14 @@ class GStreamingExecutor final
     explicit GStreamingExecutor(std::unique_ptr<ade::Graph> &&g_model,
                                 const cv::GCompileArgs &comp_args);
     ~GStreamingExecutor();
-    void setSource(GRunArgs &&args);
-    void start();
-    bool pull(cv::GRunArgsP &&outs);
-    bool pull(cv::GOptRunArgsP &&outs);
-    std::tuple<bool, cv::util::variant<cv::GRunArgs, cv::GOptRunArgs>> pull();
-    bool try_pull(cv::GRunArgsP &&outs);
-    void stop();
-    bool running() const;
+    void setSource(GRunArgs &&args) override;
+    void start() override;
+    bool pull(cv::GRunArgsP &&outs) override;
+    bool pull(cv::GOptRunArgsP &&outs) override;
+    PyPullResult pull() override;
```

Comment:
Since `GStreamingExecutor` isn't exposed to python directly (only via `GStreamingCompiled`) it doesn't affect python, because `PyPullResult` matches to this API: https://github.com/opencv/opencv/blob/4.x/modules/gapi/include/opencv2/gapi/gstreaming.hpp#L277

Reply:
Python parser knows how to parse ` std::tuple<bool, cv::util::variant<cv::GRunArgs, cv::GOptRunArgs>>`, but don't know anything about `PyPullResult` which isn't used.